deeplabcut/create_project/add.py

```diff
@@ -99,8 +105,12 @@ def add_new_videos(config, videos, copy_videos=False, coords=None):
             cfg["video_sets"].update(params)
         else:
             cfg["video_sets_original"].update(params)
+    videos_str = [str(video) for video in videos]
+    if extract_frames:
+        frame_extraction.extract_frames(config, userfeedback=False, videos=videos_str)
 
     auxiliaryfunctions.write_config(config, cfg)
+
     print(
-        "New video was added to the project! Use the function 'extract_frames' to select frames for labeling."
+        "New videos were added to the project and frames have been extracted for labeling!"
```

Comment:
You might want to keep the new print statement within the `if` and put the original print statement in an `else` (so the message is correct depending on the bool value)

Reply:
Is this fixed? If I'm not mistaken now the function will print `New videos were added to the project and frames have been extracted for labeling` even if `extract_frames` is False.

If it is false it should still print `New videos were added to the project! Use the function 'extract_frames' to select frames for labeling` no?

Or am I missing something?